tests/IabVast3Test.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Sokil\Vast;
+
+class IabVast3Test extends AbstractTestCase
+{
+    public function testWrapperTag()
+    {
+        $factory = new Factory();
```

Comment:
Interesting idea. 
But i don't think that fail of this test must fail CI because:
1) library support not all protocol
2) Change of iab/VAST 3.0 samples/No_Wrapper_Tag-test.xml will fail this test

Maybe better will be to make standalone test that run in CI or manuallty that iterate over all tests from examples (maybe on bash, or using phpunit's dataProvider annotation) and write what test fails?

And what reason to push incomplete test  to pull request? )

Reply:
Reason for incomplete test is simple - to start a conversation regarding:
- found issues(missing implementations)
- way I done "integration" with IAB sample VASTS

If lib could be tested against IAB examples it would be lib maturity and quality sign. But I don't see a good way to implemented. Current implementation, as you properly mentioned is fragile. As an option, I probably could point to some commit at IAB examples repo, thus locking "version" for examples, but it's not clear how to update this dependency. But even if it will frozen it still could benefit to maturity and quality. Another option would be copy examples from original repo to lib repo, but this way doesn't feel right.